### ISSUE TITLE
Put bitCNY At The Top Of The List in Margin Positions

### ISSUE THREAD

**Issue by landry314** [bitshares/bitshares-ui#1098]
Since bitCNY has a much higher volume than bitBTC, I think it should be at the top of the Margin Positions list by default.

**Comment by wmbutler** [comment]
Sort order default to volume? I'm not sure most ppl would agree. Seems like it should default to highest margin positions for that user. I never margin CNY, so I wouldn't care necessarily.

**Comment by nbcks** [comment]
Maybe some way for the user to customize the order they see? So default sort by volume, then allow the user to change the ordering in some way?

**Comment by landry314** [comment]
Good Idea, @nbcks!

**Comment by landry314** [comment]
Yes, highest margin positions for that user, if they have any open, otherwise, sort the rest by volume.

**Comment by startailcoon** [comment]
I see two ways. 
- making these "starred" as well, like markets or accounts
- freely sorted which would mean implementing something completely new

**Comment by wmbutler** [comment]
There are very few items in Margin Positions. Descending sort on either the Total or Debt columns should be sufficient.

**Comment by nbcks** [comment]
I think @wmbutler is right. Also sorting by volume would require getting the volume, which is slow as it requires yet another poll to the api server. Sorting by total or debt column seems the most reasonable course of action.

**Comment by wmbutler** [comment]
Anyone claiming this?

**Comment by nbcks** [comment]
Could I claim it please?

**Comment by nbcks** [comment]
Not sure I actually have enough time to do this so I think I'm going to have to pass on this issue :(

**Comment by wmbutler** [comment]
Thanks for letting me know.

**Comment by AustinAmoruso** [comment]
I'll take this one

**Comment by wmbutler** [comment]
Assigned for the next Sprint.

**Comment by wmbutler** [comment]
@AustinAmoruso please release this issue if you aren't going to work it.

**Comment by AustinAmoruso** [comment]
Release this issue I will not have time to get to this sorry for being so behind.

**Comment by startailcoon** [comment]
I'll take this

**Comment by startailcoon** [comment]
So, do I understand this correct, that we wish the bitCNY asset at the top of the list always?
If it's not borrowed, I'd advice it to be below any borrowed asset at least, so the list would be the following:
- Assets shorted by the user
- bitCNY
- The rest of default listed assets.

I can't say that I understand why we don't want it listed by name, but I'm not a big shorter so there may be good cases for this.

**Comment by sschiessl-bcp** [comment]
Thus definitely goes over the assigned hour, but maybe we can use this to implement a generally sortable table list component where every column allows click to sort with proper store of the sorted table in SettingsStore

**Comment by startailcoon** [comment]
@svk31 has done a wrapper function for Margin Assets since a few days ago, that I can't exactly figure out how its working.

https://github.com/bitshares/bitshares-ui/blob/66678211f96b6b5e1f8c842092997d3996c314d7/app/components/Account/MarginPositions.jsx#L644-L646

**Comment by startailcoon** [comment]
I'm releasing this issue for now

**Comment by svk31** [comment]
There was a bug in the AssetWrapper class with the defaultProps, and I hadn't updated MarginPositions properly either. Fixed now

**Comment by dot5enko** [comment]
@wmbutler fixed it by accident, so can you assign it to me?

**Comment by startailcoon** [comment]
Nice work @dot5enko 👍
